x-pack/legacy/plugins/uptime/public/uptime_app.tsx

```diff
@@ -100,7 +100,7 @@ const Application = (props: UptimeAppProps) => {
                       <EuiPage className="app-wrapper-panel " data-test-subj="uptimeApp">
                         <main>
                           <PageHeader setBreadcrumbs={setBreadcrumbs} />
-                          // @ts-ignore we need to update the type of this prop
+                          {/* @ts-ignore we need to update the type of this prop*/}
```

Comment:
```suggestion
                          {/* @ts-ignore TODO we need to update the type of this prop*/}
```

While we're here, let's put a `TODO` string in there so if someone is searching for TODOs this will show up.